test/Exception/FeatureNotAvailableExceptionTest.php

```diff
@@ -12,21 +13,30 @@
 
 class FeatureNotAvailableExceptionTest extends TestCase
 {
-    const FEATURE_NAME = 'featureName';
+    private const FEATURE_NAME = 'featureName';
 
+    /**
+     * @return void
```

Comment:
Is there a reason the void return type is in a comment and not written as:
`public function testPassingAnEmptyMessageTriggersMessageBuilding(): void`?

Reply:
I think I copied it from somewhere, but it is neater with the actual code syntax, so I've swapped them all over. I'll do it in the other repos as well.